src/palace/manager/celery/tasks/collection_delete.py

```diff
@@ -40,6 +40,6 @@ def run(self) -> None:
             collection.delete()
 
 
-@shared_task(queue=QueueNames.high, bind=True)
+@shared_task(queue=QueueNames.high, bind=True, task_ignore_results=True)
```

Comment:
I don't see `task_ignore_results` anywhere in the Celery documentation. Is that the parameter that we need to be passing here? From the docs it looks like `ignore_result` is what we want to pass here https://docs.celeryq.dev/en/stable/reference/celery.app.task.html#celery.app.task.Task.ignore_result

Reply:
Ah: that explains a lot.  I'm going to try setting task_ignore_result=True in the config and remove all of what should have been ignore_result.   Hopefully that will give us the default behavior we want with the backend enabled.